### DISCUSSION TITLE
./mvnw command not found

### DISCUSSION THREAD

**Discussion by wangzitom12306** [pmd/pmd#4866] [Q&A] [answered]
When I try to compile PMD using the command ` ./mvnw clean verify`, it shows `./mvnw: command not found`.
I don't know what to do to solve this error. Any help will be greatly appreciated.

```
[abc@xyz pmd-src-7.0.0-rc4]$ ./mvnw clean verify
./mvnw: command not found
```

**Comment by adangel** [comment, marked as answer]
That's a small bug. The file is there, but not executable. Just run once `chmod a+x ./mvnw`, then it should work.

**Reply by wangzitom12306** [reply]
I run `chmod a+x ./mvnw`, it works! 
Thank you very much, really appreciated.
